src/components/vr_cooperation/include/vr_cooperation/commands/json_notification.h

```diff
@@ -30,8 +30,8 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_BASE_COMMAND_NOTIFICATION_H_
-#define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_BASE_COMMAND_NOTIFICATION_H_
+#ifndef SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_JSON_NOTIFICATION_H_
+#define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_JSON_NOTIFICATION_H_
 
 #include "application_manager/service.h"
 #include "functional_module/function_ids.h"
```

Comment:
Because I use MobileFunctionID in SendNotification(MobileFunctionID, Json::Value)